File: programmers/기지국 설치/solve.js
function solution(n, stations, w) {
    var answer = 0;
    
    for(let i = n; i > 0; --i) {
        if(i <= stations[stations.length - 1] + w && i >= stations[stations.length - 1] - w) {
            i = stations[stations.length - 1] - w;       
            stations.pop();
        }
        else {
            ++answer;
            i -= 2 * w;
        }
    }
    return answer;
}

File: programmers/올바른 괄호/solve.cpp
#include <stack>
#include <string>
#include <iostream>

using namespace std;

bool solution(string s)
{
    ios::sync_with_stdio(0);
    cin.tie(0);
    bool answer = true;
    stack<char> tmp;
    
    for(int i = 0, size = s.size(); i < size; ++i)
    {
        if (s[i] == ')')
        {
            if(tmp.empty())
                return false;
            tmp.pop();
        } else if (s[i] == '(')
            tmp.push('(');
    }
    if (!tmp.empty())
        return false;
    return answer;
}